course/templates/course/index.html

```diff
@@ -31,33 +31,31 @@ <h2 class="panel-heading">{% trans "My courses" %}</h2>
 
 <div class="frontpage-panel">
 	<h2 class="panel-heading">{% trans "All ongoing courses" %}</h2>
-	<div class="row frontpage">
+	<section class="frontpage frontpage-section">
 		{% site_advert as advert %}
 		{% if advert %}
-			<div class="col-sm-6 col-md-4 col-lg-3">
+			<div class="frontpage advert-card card">
 				<a href="{{ advert.href }}" target="_blank">
-					<div class="card advert-card">
-						<div class="advert-image" aria-hidden="true">
-							{% if advert.image %}
-								<img class="card-img-top" src="{{ advert.image }}" alt="" />
-							{% else %}
-								<div class="card-img-top">
-									<i class="glyphicon glyphicon-comment"></i>
-								</div>
-							{% endif %}
-						</div>
-						<div class="card-body">
-							<h3 class="card-title">{{ advert.title }}</h3>
-							<p class="card-text">{{ advert.text|safe }}</p>
-						</div>
+					<div class="advert-image" aria-hidden="true">
+						{% if advert.image %}
+							<img class="card-img-top" src="{{ advert.image }}" alt="" />
+						{% else %}
+							<div class="card-img-top">
+								<i class="glyphicon glyphicon-comment"></i>
+							</div>
+						{% endif %}
+					</div>
+					<div class="card-body">
+						<h3 class="card-title">{{ advert.title }}</h3>
+						<p class="card-text">{{ advert.text|safe }}</p>
```

Comment:
Does this behave differently than the other courses? Why?. Btw, how do you mark a course as  `advert`?. I was trying to find this option, but I could not find it.

Reply:
You mean the advert? That whether that behaves differently? Yeah, it was a bit hard to find at first for me too. But there is a sidewide advertisement that you can place from settings.py and it shows like this on the frontpage. So acts like as one of the course-cards:

![image](https://user-images.githubusercontent.com/43036333/87391167-ea102a80-c5b2-11ea-9a09-652ca40775b5.png)

Teachers can't set that for any course basically, it is only for the site-managers to set.

Reply:
Great, I did not know about that option :smile: . And yes, I was asking about the advert card.